services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/ConverterUtils.java

```diff
@@ -128,55 +128,52 @@ private static int findTrimExclusiveEnd(String string, char characterToTrim, int
         }
     }
 
-    public static String[] chunk(String valueToChunk, int... splitSizes) {
-        String[] result = new String[splitSizes.length + 1];
-        int splitStartInclusive = chunkLeft(valueToChunk, result, splitSizes);
+    public static String[] chunkDateTime(String valueToChunk) {
```

Comment:
Can we have little javadocs on these so it's easier for me to assert they are doing what they are supposed to be doing.